src/app/history/history-bet/history-bet.component.css

```diff
@@ -0,0 +1,16 @@
+.history-container {
+  min-height: 120px;
+
+  background-color: var(--background);
+}
+
+.history-title {
+  display: flex;
```

Comment:
why title is flex? isn't it an overhead?

Reply:
done

Reply:
done